src/cljdoc/render/articles.clj

```diff
@@ -33,12 +33,17 @@
    (when (seq doc-bundle)
      (->> doc-bundle
           (map (fn [doc-page]
-                 (let [slug-path (-> doc-page :attrs :slug-path)]
+                 (let [slug-path (-> doc-page :attrs :slug-path)
+                       external-url (-> doc-page :attrs :cljdoc.doc/external-url)]
                    [:li
                     {:class (when (seq (:children doc-page)) "mv2")}
+                    (when external-url
+                      [:img.v-mid.mr1 {:src "https://microicon-clone.vercel.app/external/12"}])
                     [:a.link.blue.hover-dark-blue.dib.pv1
                      {:style {:word-wrap "break-word"}
-                      :href  (doc-link version-entity slug-path)
+                      :target "_blank"
+                      :rel "noopener"
```

Comment:
Oh, shoot, I wasn't very explicit here. We only want `:target "_blank" :rel "noopener"` when there's an external-url.

Reply:
Ok I did a bunch of research to see what the deal was here and I'm on-board with this now, just remove the `:target` and `:rel` imo. Sorry for the runaround!